portfolio.css

```
@@ -0,0 +1,90 @@
+@import url("./normalize.css");
+@import url('https://fonts.googleapis.com/css?family=Lobster&display=swap');
+@import url('https://fonts.googleapis.com/css?family=PT+Sans&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Oxanium&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Spartan&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');
+* {
+	margin: 0;
+}
+
+
+
+html{
+	font-size: 62.5%;
+}
+header {
+	background-image: url(assets/mountains.jpeg);
```

Personal preference, but I think it's nicer to wrap the path in quotes.
```suggestion
	background-image: url("assets/mountains.jpeg");
```